File: Lista3/zad1/Szesciakat.java
public class Szesciakat extends Figura{
    double a;
    public Szesciakat(double a)
    {
        this.a = a;
    }
    public double obliczObwod() {
        return a*6;
    }
    
    public double obliczPole() {
        return 3*a*a*Math.sqrt(3)/2;
    }

    public String podajNazwe()
    {
        return "Szesciokat";
    }
}

File: Lista3/zad2/Szesciakat.cpp
#include "Szesciakat.hpp"
#include <math.h>
#include <string>

double Szesciakat::obliczPole()
{
    return 3*bok*bok*sqrt(3)/2;
}

double Szesciakat::obliczObwod()
{
    return bok*6;
}

Szesciakat::Szesciakat(double bok)
{
    this->bok = bok;
}

std::string Szesciakat::podajNazwe()
{
    return "Szesciokat";
}
